Service/ExpressPay/QuoteConverter.php

```diff
@@ -167,6 +167,13 @@ static function (Rate $rate) use (&$usedRateCodes): bool {
         $hasRequiredAddressData = ($shippingAddress->getCity() && $shippingAddress->getCountryId());
 
         if ($hasRequiredAddressData && count($shippingRates) > 0) {
+            $validShippingRates = array_filter(
+                $shippingRates,
+                static function (Rate $rate): bool {
+                    return substr($rate->getCode(), -strlen('_error')) !== '_error';
```

Comment:
Opted _not_ to use `str_ends_with()` here to remain backwards compatible with php 7.

Reply:
Nice idea, I like it!